rfhub2/cli/statistics/statistics_extractor.py

```diff
@@ -95,8 +95,16 @@ def parse_xml_keywords(self) -> List[XmlKeyword]:
             )
             for xml_keyword in xml_keywords
             if xml_keyword.attrib.get("library") is not None
+            and self.has_keyword_passed(xml_keyword)
         ]
 
+    @staticmethod
```

Comment:
Could you expand the docstring to include why this change is needed in 4.x and confirm whether it affects rfhub when it's run with robotframework < 4.0?

Reply:
@pbylicki sure. This is not exactly needed by rfwk 4+, but would be a safeguard, to not include failed keywords in statistics gathering. I will highlight that in docstring